### PR TITLE
Make Sections Collapsible

### LINKED ISSUE #21: Collapse sections
Mentioned in video

### PR DESCRIPTION
**Pull request by lukepk**
This resolves #21 

Some of the sections have subsections (named categories in the code). Do we want these to also be individually collapsible?

### CODE REVIEW

**Comment by lukepk** [comment]
Breaks the autoreader as it nests the text another div deeper. Related to issue #19. Will probably fix that issue first before coming back to this.

**Comment by Muggyfox1** [comment]
I like the idea of collapsible subsections. I tried out what you've done so far, works great!

File: guide.css [hunk 1]
```diff
@@ -10,9 +10,35 @@ img{
...
+}
+
+/* Collapsible section content. Note: visible by default */
+.collapsible-content {
+	display: inline;
+	overflow: hidden;
+}
+
+.active:after {
+	content: '\02795'; /* Unicode character for "plus" sign (+) */
```
Comment (MichaelEbert) [inline guide.css:41]: Just insert the Unicode character directly, no need to encode it

File: guide.css [hunk 2]
```diff
@@ -10,9 +10,35 @@ img{
...
+	width:100%;
+}
+
+.sectionTitle:hover {
+	background-color: #ccc;
+}
+
+.sectionTitle:after {
+	content: "\2796"; /* Unicode character for "minus" sign (-) */
+	font-size: 13px;
```
Comment (MichaelEbert) [inline guide.css:28]: stick with em or relative sizes instead of px

File: guide.css [hunk 3]
```diff
@@ -10,9 +10,35 @@ img{
...
+	border:none;
+	cursor:pointer;
 	text-align: center;
 	font-size: 1.5em;
 	border-bottom: 2px solid gray;
+	width:100%;
+}
+
+.sectionTitle:hover {
+	background-color: #ccc;
```
Comment (MichaelEbert) [inline guide.css:23]: 6-digit color codes plz

File: guide.css [hunk 4]
```diff
@@ -10,9 +10,35 @@ img{
 }
 
 .sectionTitle{
+	background-color: #FBEFD5;
+	border:none;
```
Comment (MichaelEbert) [inline guide.css:14]: border:  none is redundant and can be removed i think

File: collapsible.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+/** 
+ * Handles making sections of the speedrun route collapsible.
+ */
+
+// Make sections collapsible
+function makeCollapsible() {
+    let coll = document.getElementsByClassName("sectionTitle")
+    for(let i = 0; i < coll.length; i++) {
```
Comment (MichaelEbert) [inline collapsible.js:8]: is there a reason we're using indexing syntax instead of iterator syntax? (for(x of y))

File: guide.css [hunk 6]
```diff
@@ -10,9 +10,35 @@ img{
...
+.sectionTitle:after {
+	content: "\2796"; /* Unicode character for "minus" sign (-) */
+	font-size: 13px;
+	color: white;
+	float: right;
+	margin-left: 5px;
+}
+
+/* Collapsible section content. Note: visible by default */
+.collapsible-content {
```
Comment (MichaelEbert) [inline guide.css:35]: All other compound titles are in camelCase, we should probably keep that theme

File: collapsible.js (hunk 5 above)
Comment (lukepk) [reply collapsible.js:8]: Wrote it this way to maintain browser compatibility for IE. Happy to change to for/of if you'd prefer to stick to ES6.

File: guide.css (hunk 6 above)
Comment (lukepk) [reply guide.css]: Changed in 7e6afadfff67b96118a4411de22f7e26c54a3d34

File: guide.css (hunk 4 above)
Comment (lukepk) [reply guide.css]: Removed in cfd429c18c1e60c75045ad0874b44a72bfe6769b

File: guide.css (hunk 3 above)
Comment (lukepk) [reply guide.css]: Done

File: guide.css (hunk 2 above)
Comment (lukepk) [reply guide.css]: Done

File: guide.css (hunk 1 above)
Comment (lukepk) [reply guide.css]: Done

File: collapsible.js (hunk 5 above)
Comment (MichaelEbert) [reply collapsible.js:8]: Yeah I'm already using Fetch, CSS3, etc., so backwards compatibility is right out.

Comment (MichaelEbert) [reply collapsible.js:8]: my targets are firefox 92, chrome (whatever's current), and the mobile equivalents

Comment (lukepk) [reply collapsible.js:8]: Alright noted for the future. Will leave as is for now since its been merged.

**Pull request merged**